src/app/components/context-menu/context-menu-outside-event.directive.ts

```diff
@@ -0,0 +1,29 @@
+import { Directive, Output, EventEmitter, OnInit, OnDestroy } from '@angular/core';
+import { fromEvent, Subscription } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Directive({
+  selector: '[acaContextMenuOutsideEvent]'
+})
+
+export class OutsideEventDirective implements OnInit, OnDestroy {
+    private subscriptions: Subscription[] = [];
+
+    // tslint:disable-next-line:no-output-rename
+    @Output('clickOutside') clickOutside: EventEmitter<null> = new EventEmitter();
```

Comment:
You can use empty `@Output()` if the names are the same. That will also remove the need for tslint hack.